content/en/docs/components/katib/katib-config.md

```diff
@@ -6,436 +6,393 @@ weight = 70
-   the suggestion's deployment must have permission to update the experiment's
-   trial status. If you don't specify a service account in the Katib config,
-   Katib controller creates required
-   [Kubernetes Role-based access control](https://kubernetes.io/docs/reference/access-authn-authz/rbac)
-   for the suggestion.
-
-   If you need your own service account for the experiment's
-   suggestion with early stopping, you have to follow the rules:
-
-   - The service account name can't be equal to
-     `<experiment-name>-<experiment-algorithm>`
-
-   - The service account must have sufficient permissions to update
-     the experiment's trial status.
-
-### Suggestion volume settings
-
-When you create an experiment with
+### Metrics Collectors Parameters
+
+Parameters set in `.runtime.metricsCollectors` configure container for
+[the Katib metrics collector](docs/components/katib/experiment/#metrics-collector).
+The following settings are **required** for each Katib metrics collector that you want to use in your Katib Experiments:
+
+- `kind` - one of the Katib metrics collector types.
+
+- `image` - a Docker image for the metrics collector's container.
+
+The following settings are **optional**:
+
+- `imagePullPolicy` - an [image pull policy](https://kubernetes.io/docs/concepts/configuration/overview/#container-images)
+  for the metrics collector's container.
+
+  The default value is `IfNotPresent`
+
+- `resources` - [resources](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#resource-requests-and-limits-of-pod-and-container)
+  for the metrics collector's container.
+
+  The default values for the `resources` are:
+
+  ```yaml
+  metricsCollectors:
+    - kind: StdOut
+      image: docker.io/kubeflowkatib/file-metrics-collector:latest
+      resources:
+        requests:
+          cpu: 50m
+          memory: 10Mi
+          ephemeral-storage: 500Mi
+        limits:
+          cpu: 500m
+          memory: 100Mi
+          ephemeral-storage: 5Gi
+  ```
+
+  You can run your metrics collector's container without requesting
+  the `cpu`, `memory`, or `ephemeral-storage` resource from the Kubernetes cluster.
+  For instance, you have to remove `ephemeral-storage` from the container resources to use the
+  [Google Kubernetes Engine cluster autoscaler](https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-autoscaler#limitations).
+
+  To remove specific resources from the metrics collector's container set the
+  negative values in requests and limits in your Katib config as follows:
+
+  ```yaml
+  resources:
+    requests:
+      cpu: -1
+      memory: -1
+      ephemeral-storage: -1
+    limits:
+      cpu: -1
+      memory: -1
+      ephemeral-storage: -1
+  ```
+
+- `waitAllProcesses` - a flag to define whether the metrics collector should wait until all
+  processes in the Trial's training container are finished before start to collect metrics.
+
+  The default value is `true`
```

Comment:
This parameter's default is nil, which means `false`.: 

- https://github.com/kubeflow/katib/blob/19268062f1b187dde48114628e527a2a35b01d64/pkg/apis/config/v1beta1/types.go#L125
- https://github.com/kubeflow/katib/blob/19268062f1b187dde48114628e527a2a35b01d64/pkg/apis/config/v1beta1/defaults.go#L167